src/input.rs

```diff
@@ -16,12 +16,14 @@ struct State {
     keys_pressed: HashSet<Key>,
     mouse_pressed: HashSet<MouseButton>,
     mouse_pos: mint::Point2<f32>,
+    mouse_pos_ndc: mint::Point2<f32>,
```

Comment:
not sure why we'd want to have NDC and non-NC both tracked and exposed? The users can convert, if needed

Reply:
Converting needs access to window size. There are several ways to do so: 
1. Store window size inside input and convert to ndc on demand. 
2. Pass Renderer to Orbit::update
3. Move map_to_ndc function outside Renderer and pass window size to it as parameter

Reply:
It's a small and useful feature that's hardly going to be a performance bottleneck. Why burden the user with a manual conversion?

Reply:
It's needed for ray casting, for example.

Reply:
Agreed with alteous. It simplifies the API by holding all related stuff in one place.